test/Chirp.CLI.Client.Tests/UnitTests.cs

```diff
@@ -1,10 +1,23 @@
+using System.Text.RegularExpressions;
+using System.Xml.XPath;
+using Chirp.CLI.Client;
+using Xunit;
+
 namespace Chirp.CLI.Client.Tests;
 
 public class Tests
 {
+    //UNITTEST -> Test that the UNIX time is converted to a string correctly
```

Comment:
Unnecessary comment for test method.
Usually, we don't comment what a test method tests, as it should be easy to understand from its name. Therefore, it is allowed, and nearly recommended, to have more verbose test method names.
I'd thus recommend you make the method name a tad more verbose:)